recipes/busybox/busybox-1.24.1/defconfig

```diff
@@ -759,7 +823,7 @@ CONFIG_FEATURE_IFUPDOWN_IP=y
 CONFIG_FEATURE_IFUPDOWN_IPV4=y
 # CONFIG_FEATURE_IFUPDOWN_IPV6 is not set
 # CONFIG_FEATURE_IFUPDOWN_MAPPING is not set
-CONFIG_FEATURE_IFUPDOWN_EXTERNAL_DHCP=y
+# CONFIG_FEATURE_IFUPDOWN_EXTERNAL_DHCP is not set
```

Comment:
Why are you changing this?  I think you are causing breakage in the busybox_udhcp USE flag.